python/paddle/fluid/dygraph/dygraph_to_static/loop_transformer.py

```diff
@@ -106,23 +113,65 @@ def get_loop_var_names(self, node):
         return loop_var_names, create_var_names
 
     def visit_Name(self, node):
+        if self._is_call_func_name_node(node):
+            self.generic_visit(node)
```

Comment:
👍🏻👍🏻
Whether `gast.Name` is a leaf node in AST.  If does, it's no need to call `self.generic_visit`. I'm not sure for that and it's just a small question.😁

Reply:
Considering gast.Name has annotation as gast.Expr, let's keep visiting.

![image](https://user-images.githubusercontent.com/7913861/76393554-b9bf4200-63ae-11ea-93f0-d1a1d5eaa9cc.png)